data/mods/gen1/moves.ts

```diff
@@ -904,6 +905,7 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		name: "Substitute",
 		pp: 10,
 		priority: 0,
+		selfDamage: "25% (can't KO)",
```

Comment:
It's definitely weird that some of these are implemented by the engine, and some of these are scripted.

I guess it's fine to just do this for now and add engine support for them in the future.